test/parallel/test-util-inherits.js

```diff
@@ -75,6 +75,9 @@ assert.strictEqual(e.e(), 'e');
 assert.strictEqual(e.constructor, E);
 
 // should throw with invalid arguments
-assert.throws(function() { inherits(A, {}); }, TypeError);
-assert.throws(function() { inherits(A, null); }, TypeError);
-assert.throws(function() { inherits(null, A); }, TypeError);
+assert.throws(function() { inherits(A, {}); },
+              common.expectsError('ERR_NO_PROTOTYPE', TypeError));
```

Comment:
The signature for `expectsError()` changed recently. You'll want this to be `common.expectsError({code: 'ERR_NO_PROTOTYPE', type: TypeError});`

See doc for the function in test/README.md.